### PR TITLE
Awesome counts

### LINKED ISSUE #265: Awesome-count is broken in multiple places
Broken on both /random and u/lionleaf/awesome (and I think that should be it)

@sigvef

### PR DESCRIPTION
**Pull request by sigvef**
This fixes #265 .

### CODE REVIEW

File: dwitter/feed/views.py [hunk 1]
```diff
@@ -92,6 +91,12 @@ def feed(request, page_nr, sort):
         .select_related('reply_to__author__username')
         .prefetch_related('comments'))
 
+    # For some reason order_by('?') and .annotate(num_likes=Count('likes'))
+    # don't work together, so we need to do extra work if the sorting is random.
```
Comment (lionleaf) [inline dwitter/feed/views.py]: This line is a single character too long, so the lint and build fails. FIX IT! :)

Comment (iver56) [reply dwitter/feed/views.py]: The 80 character limit should not be a hard limit IMO. Fun fact: The 80 character "limit" is based on the legacy of the 80 column punched card format

Comment (lionleaf) [reply dwitter/feed/views.py]: While I tend to agree I also think it's alright to have a strict build system based on the linter. We could change the lint config though

**Comment by sigvef** [comment]
I reformatted the comment that tripped up the build. Since you've already approved and there's no other changes, I'll go ahead and click merge on this :)

**Pull request merged**